### ISSUE TITLE
Screen Reader Accessibility

### ISSUE THREAD

**Issue by Menelion** [AdguardTeam/AdGuardVPNForWindows#79]
### Prerequisites

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Issue Details

- **AdGuard VPN version:** 1.0 Beta 1
- **VPN mode:** N/A
- **Operating system and version:** Windows 10 Version 20H2 (OS Build 19042.985)
- **Other network-level software:** N/A

### Expected Behavior
The VPN software should be accessible for blind users using screen reading software. According to other users' reports, it is fully accessible under iOS, but on Windows the accessibility is nearly absent.  
Tested with JAWS for Windows 2021 (today's update), NVDA latest snapshot, and Narrator.

### Actual Behavior
1. There is no way to access settings. The Settings button is accessible by pressing Tab and correctly labeled, but there is no possibility to actually see what is in the settings. Even more advanced techniques like JAWS cursor, touch cursor or object review can give only a glimpse of information, like the presence of a General group of settings. Even when selecting Settings in the system tray menu (which is fully accessible, fortunately), the same totally inaccessible window is presented.
2. when on the first screen after installation, the fields for email and password are not labeled, plus there are some unlabeled (perhaps hidden) buttons. The Sign Up and Sign In buttons are correctly labeled and accessible.
3. The password field does not convey information about its type (i.e., the screen reader should say "password edit", and not just "edit").
4. when selecting Other Servers from the system tray menu, the list (or tree view, or whatever it is — sorry, I can't tell) is completely inaccessible.
5. When pressing the Exclusions button, the buttons (like Add Website, Export/Import Exclusions) are accessible, but the rest of the window is not: there is an unlabeled edit box, a set of unlabeled radio buttons and some unlabeled buttons.

### Screenshots
N/A

### Additional Information

Generally speaking, AdGuard products have always been accessible for blind users and they are loved among the blind community, especially the Russian-speaking one, and the efforts of your team are hugely appreciated (you won't believe how many developers the blind users are angry at). As far as the AdGuard VPN is concerned, I love the product for its lightness on CPU resources, for the blazing fast VPN and for the choice of servers in different countries. However if I decide to pay for it, I'd like to benefit from the full capacity of the software. Most of the accessibility quirks are due to absence of correct labeling (I especially thank you for using native widgets or widgets that appear native to screen readers — I mean buttons and radio buttons: thus fixing the Exclusions dialog would take like half an hour). The harder thing to fix would be the inaccessibility of whole parts like Settings or Other Servers selection.  
I'm ready to help in testing: I have a long experience of alpha and beta testing various software, I'm a software engineer myself — and also I speak Russian 😁.

**Comment by Aydinv13** [comment]
@Menelion Thank you, it is very nice that you appreciate our product! Yes, it's a crucial part and we'll work on it so it will be definitely implemented in the nearest future.

**Comment by yastrebilya2018** [comment]
Hello.  Hopefully these issues will be fixed.  Your company is doing very well in accessibility and your products are readable by screen readers.  All of the above is true for me as well.  The same problems were in your first beta version.  It's okay, we'll wait for a fix.  Thank you for making the service available to everyone!

**Comment by Anteecs** [comment]
One of AdGuard users reached out to AdGuard Support Team with the following: "Some
elements of the AdGuard VPN program are not voiced by NVDA. For example, I cannot
listen to the remaining traffic - it's just not voiced. Also in the settings, I can’t enable or disable something - it also doesn’t work: I press the Space button and nothing happens, I can not select a DNS server - is also not activated. 
Please make it work with NVDA! After all, the AdGuard blocker works fine with NVDA!
Thanks in advance!"
Could those features also be added in the next release of AdGuard VPN Windows app?
Thank you.

**Comment by Ancientlizard** [comment]
Hello!
As for voicing the amount of remaining traffic, a very good remark!
With regards to the broken buttons in the settings, the buttons work, but screen readers are not given information about the state of the element.
for example
on/off.
Otherwise, everything works properly!
But of course, if these problems are fixed, it will be great!

**Comment by maxikuzmin** [comment]
@Ancientlizard we are looking into this issue
